packages/primitives/tabs/src/tabs-content.directive.ts

```diff
@@ -0,0 +1,20 @@
+import { computed, Directive, inject, input } from '@angular/core';
+
+import { TABS_CONTEXT_TOKEN } from './tabs-context.service';
+
+@Directive({
+    selector: '[TabsContent]',
+    standalone: true,
+    host: {
+        role: 'tabpanel',
+        tabindex: '0',
+        '[hidden]': '!selected()'
```

Comment:
this could probably cause animation problems

Reply:
Yes, you're right, there may be difficulties with Animation transition. I will return to this part pf code after the main functionality.